src/gui/accountstate.cpp

```diff
@@ -120,6 +128,10 @@ void AccountState::setState(State state)
         if (oldState == Connected || _state == Connected) {
             emit isConnectedChanged();
         }
+        if (_state == Connected) {
+            _checkConnectionTimer.setInterval(ConnectionValidator::DefaultCallingIntervalMsec);
```

Comment:
@mgallien Why is this `_checkConnectionTimer.setInterval(ConnectionValidator::DefaultCallingIntervalMsec);` needed here? You've already set this in the construtor and don't change in other places.

Reply:
leftover from previous iterations
going to remove it